test/C/test_read.jl

```diff
@@ -25,16 +25,16 @@ using Gurobi, Test
 
     simple_model = Gurobi.Model(simple_model_env, "simple_mip", :maximize)
 
-    add_ivar!(simple_model, 0., 0, Inf)  # p1
-    add_ivar!(simple_model, 0., 0, Inf)  # p2
-    add_ivar!(simple_model, 0., 0, Inf)  # p3
-    add_cvar!(simple_model, 1., 0., Inf) # z
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p1
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p2
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p3
+    add_cvar!(simple_model, 1.,  0.0, Inf) # z
     update_model!(simple_model)
 
-    add_constr!(simple_model, [3., 5., 4., -1.], '<', 0.)
-    add_constr!(simple_model, [3., 5., 4., -1.], '>', 0.)
+    add_constr!(simple_model, [3., 5., 4., -1.], '<', 0.0)
+    add_constr!(simple_model, [3., 5., 4., -1.], '>', 0.0)
 
-    add_constr!(simple_model, [0.5, 2., 1., 0.], '<', 6.)
+    add_constr!(simple_model, [0.5, 2., 1., 0.0], '<', 6.)
 
     setparam!(simple_model, "Heuristics", 0.0)
     setparam!(simple_model, "Presolve", 0)
```

Comment:
I used the same model for testing that I used for another function - I also changed all "0." values to "0.0". Would it be better to make a single test file that contains a test sets for either function?

Reply:
No, duplication in tests doesn't matter.